test/bigtest/tests/create-order-test.js

```diff
@@ -53,4 +58,16 @@ describe('Create order', () => {
       expect(form.suffixSelect.value).to.be.equal('');
     });
   });
+
+  describe('Create new order', () => {
+    beforeEach(async () => {
+      await form.orderTypeSelect.select(ORDER_TYPE.oneTime);
+      await form.vendorInput.fill('EBSCO');
+      await form.createOrderButton.click();
+    });
+
+    it('dispalys list of orders, new order is created ', () => {
+      expect(orders.$root).to.exist;

```

Comment:
the thing is that it always display a list of orders. I think you also need to add a check that form is not displayed anymore `form.isPresent` to be `false`.